tests/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -l
+
+# aloe will only work when called from the tests directory
+cd tests
+aloe features/*
```

Comment:
Does this script return a non-zero exit code if the tests fail? Otherwise, I think the green check mark shows up even if some of the tests fail. 

If this doesn't currently output an exit code, this could be helpful: https://stackoverflow.com/questions/90418/exit-shell-script-based-on-process-exit-code